src/UIComponents/PhraseBuilder.jsx

```diff
@@ -0,0 +1,143 @@
+/* React component to handle setting datatype for selected columns. */
```

Comment:
I don't think this is quite what this component handles... maybe "to handle selecting label and features" ?